CONTRIBUTING.md

```diff
@@ -1,55 +1,69 @@
+If you can help answer questions in your area of expertise, it would be a benefit to the community.
 
-
-If you really want to file a bug or request a feature, go to this [issue list](https://github.com/OpenRefine/OpenRefine/issues). Please use the search function first to make sure a similar issue doesn't already exist. 
+If a forum discussion determines there is a bug in OpenRefine or a new feature is identified,
+we welcome bug reports and feature requests. Please search the [issue tracker](https://github.com/OpenRefine/OpenRefine/issues) first to make sure
+the bug / feature hasn't already been added. Note: the development team principally works from the issue
+tracker, so anything not included there risks getting lost.
 
 ## Promote OpenRefine
 
-You don't need to be a coder to contribute to OpenRefine. Did you write a tutorial or article about OpenRefine on your blog or site? Are you organizing a workshop or presentation for OpenRefine in your city? Let us know via our [forum](https://forum.openrefine.org/) or Twitter account ([@OpenRefine](http://twitter.com/OpenRefine)). We will share the news via our monthly update and via our Twitter handle. 
+Promoting OpenRefine is a great way to give back. Did you write a tutorial or article about OpenRefine on your blog or site?
+Are you organizing a workshop or presentation for OpenRefine in your city? Let us know via our [forum](https://forum.openrefine.org/) or Twitter account ([@OpenRefine](http://twitter.com/OpenRefine)).
+We will share the news via our monthly update and via our Twitter handle.
 
-## Contributing translations
+## Contribute translations
 
-You can help us [translate OpenRefine](https://docs.openrefine.org/technical-reference/translating-ui) in as many languages as possible [via Weblate](https://hosted.weblate.org/engage/openrefine/?utm_source=widget).
+We want OpenRefine to be available in as many languages as possible to serve
+the biggest community of users. You can help us [translate OpenRefine](https://docs.openrefine.org/technical-reference/translating-ui) into languages you are fluent in [via Weblate](https://hosted.weblate.org/engage/openrefine/?utm_source=widget).
+Although we have the beginnings of translations for many languages, only a few are complete and popular languages
+like Spanish, Brazilian Portuguese, and French could use help.
 
-## Contributing documentation
+## Contribute documentation
 
-When browsing our documentation, feel free to use the edit button to suggest improvements. For large changes, you might want to [prepare your changes locally](https://openrefine.org/docs/technical-reference/contributing#contributing-to-the-documentation).
+When browsing our [user manual](https://openrefine.org/docs/) or other documentation, feel free to use the edit button to suggest improvements. For large changes, you might want to [prepare your changes locally](https://openrefine.org/docs/technical-reference/documentation-contributions).
```

Comment:
For large changes should we invite the person to discuss it with the community first by creating an issue or a thread on the forum?

Reply:
Done